tests/_runtime/test_runtime.py

```diff
@@ -1214,32 +1216,22 @@ async def test_runtime_resolution_failure(strict_kernel: Kernel) -> None:
                     cell_id="1",
                     code=textwrap.dedent(
                         """
-                    try:
-                      x = L(1)
-                    except NameError as e:
-                      E = e
-                    raise E
+                    x = L(1)
```

Comment:
Oh cool!

Reply:
I put in this unit test more to document it as a known bug- so the fact it passes correctly now is great !